app/assets/stylesheets/search.css.scss

```diff
@@ -1,46 +1,40 @@
-.page_result,
-.slack_result {
+.search_result {
+  display: block;
```

Comment:
Properties should be ordered border, border-bottom, border-radius, color, display, margin-bottom, padding